test/formatters/apheleia-ft.el

```diff
@@ -292,30 +294,36 @@ environment variable, defaulting to all formatters."
         ;; Some formatters use the current file-name or buffer-name to interpret the
         ;; type of file that is being formatted. Some may not be able to determine
         ;; this from the contents of the file so we set this to force it.
-        (rename-buffer in-file)
+        (rename-buffer (file-name-nondirectory in-file))
         (setq stdout-buffer (get-buffer-create
                              (format "*apheleia-ft-stdout-%S%s" formatter extension)))
         (with-current-buffer stdout-buffer
           (erase-buffer))
         (if (functionp command)
-            (progn
-              (setq in-temp-file (apheleia-ft--write-temp-file
-                                  in-text extension))
+            (let ((in-temp-file (apheleia-ft--write-temp-file
+                                 in-text extension)))
+              (setq in-temp-real-file in-temp-file)
```

Comment:
This should have been setting `in-temp-real-file`, which is checked later to verify that it wasn't modified by the formatter writing to the input file. Setting `in-temp-file` was doing nothing because that variable isn't ever read again.